### PR TITLE
SD-4744 Add proper indication on missing fields upon template creation

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: scripts/superdesk-templates/templates.js [hunk 1]
```diff
@@ -321,17 +322,30 @@
...
 
+                function validate(orig, item) {
+                    $scope.error = {};
+                    if (!item.template_name) {
+                        $scope.error.template_name = true;
+                    }
+                    if (!item.template_type) {
+                        $scope.error.template_type = true;
+                    }
+                    return Boolean($scope.error);
```
Comment (akintolga) [inline scripts/superdesk-templates/templates.js]: It looks to me that this line always returns true.

Comment (petrjasek) [inline scripts/superdesk-templates/templates.js]: :+1: right

**Comment by akintolga** [comment]
:+1:

**Comment by sjunaid** [comment]
:+1:

**Pull request merged**